serveur_calcul_python/utilitaires/frontend_chart_data_processing.py

```
@@ -0,0 +1,394 @@
+    Inputs:
+    - entree : a dict containing the fields which are required: cubf and id_er
+    
+    Returns:
+        - a tuple containing:
+            - prs_ids: a list of integers denoting the reg sets to use
+            - land_use_id: the land use id which you're trying to plot
+    '''
+    array_entry = entree[0]
+    land_use_id = int(array_entry.get('cubf',0))
+    if land_use_id==0 or land_use_id>9999:
+        raise ValueError('cubf mal spécifié doit être entre 1 et 9999')
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)